Gemfile

```diff
@@ -12,16 +12,20 @@ gem "stimulus-rails"
 gem "turbo-rails"
 gem "tzinfo-data", platforms: %i[ windows jruby ]
 
+group :development do
+  gem "web-console"
+end
+
 group :development, :test do
   gem "brakeman", require: false
-  gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
   gem "dotenv-rails"
   gem "rubocop-rails-omakase", require: false
 end
 
-group :development do
-  gem "web-console"
-end
-
 group :test do
+  gem "capybara"
```

Comment:
What is the purpose of this gem? Are we implementing it in this PR?

Reply:
Apu, the capybara gem is used for feature testing in Ruby applications, simulating how users interact with a web app. I don't currently need this, so removing this